build.sbt

```diff
@@ -17,21 +9,33 @@ val mvnEmbedderDeps = Seq(
   "org.codehaus.plexus" % "plexus-utils" % "3.1.0"
 )
 
-libraryDependencies ++= Seq(
-  "org.apache.maven" % "maven-embedder" % mvnVersion exclude("com.google.guava", "guava") exclude("org.codehaus.plexus", "plexus-utils"),
-  "org.apache.maven.resolver" % "maven-resolver-connector-basic" % mvnResolver,
-  "org.apache.maven.resolver" % "maven-resolver-transport-file" % mvnResolver,
-  "org.apache.maven.resolver" % "maven-resolver-transport-http" % mvnResolver,
-  "org.apache.maven.resolver" % "maven-resolver-transport-wagon" % mvnResolver,
-) ++ mvnEmbedderDeps
+lazy val pom = project.settings(
+  name := "pom-model",
+  organization := "com.typesafe",
```

Comment:
If we are changing organization name, let's not do com.typesafe.
Are you planning to publish this to Sonatype OSS?

Reply:
sbt plugins are published to Bintray under my control, so I can allow whoever to publish under whatever organization name. Sonatype OSS on the other hand requires authorization per organization name.